### PR TITLE
T/13 and Travis integration

### PR DESCRIPTION
**Pull request by pomek**
Fixes: #13.

This PR will introduce Travis for itself - https://travis-ci.org/ckeditor/ckeditor5-dev-tests.
The first test on Travis works pretty well: https://travis-ci.org/ckeditor/ckeditor5-clipboard

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -118,8 +127,20 @@ const utils = {
...
+						dir: coverageDir,
 						type: 'html'
+					},
+					// Generates "./coverage/lcov.info".
+					{
+						type: 'lcovonly',
+						subdir: '.',
+						dir: coverageDir
+					},
+					// Generates "./coverage/coverage-final.json".
```
Comment (Reinmar) [inline lib/utils.js]: Where is it used?

File: lib/utils.js [hunk 2]
```diff
@@ -118,8 +127,20 @@ const utils = {
 						type: 'text-summary'
 					},
 					{
-						dir: path.join( options.sourcePath, utils.coverageDirectory ),
+						dir: coverageDir,
 						type: 'html'
+					},
+					// Generates "./coverage/lcov.info".
```
Comment (Reinmar) [inline lib/utils.js]: It's here for CodeClimate?

Comment (Reinmar) [inline lib/utils.js]: If so, the comment should clarify this.

Comment (pomek) [inline lib/utils.js]: It is used by Code Climate. I will comment it.

File: lib/utils.js (hunk 1 above)
Comment (pomek) [inline lib/utils.js]: I will remove it. It is used by Codecov but we don't use it.

**Pull request merged**